<launch>

	<!-- Team Parameters -->
	<param name= "n" type = "int" value= "2"/>
	<param name= "robot_prefix" type = "str" value= "robot_"/>
	<param name= "log_path" type = "str" value= "/home/abuchan/projects/sim_ws/data/"/>
  <param name= "t_stop_bias" type= "double" value = "0.3"/>
  <param name= "c_lock" type= "bool" value = "True"/>
  <param name= "logger" type= "bool" value = "True"/>
	<rosparam param="robot_suffix"> ["00","01","02"] </rosparam>

	<!-- IMU Parameters -->
	<param name= "imu_hz"  type= "double" value= "20.0"/>
	<param name= "s2_g"  type= "double" value= "2.0"/>
	<param name= "s2_a"  type= "double" value= "4.0"/>
	<param name= "s2_wg"  type= "double" value= "0.002"/>
	<param name= "s2_wa"  type= "double" value= "0.002"/>
	<rosparam param="gd"> [True,False,False] </rosparam>

	<rosparam param="r0_x0"> [1.0, -1.5, 0.0] </rosparam>

  <!-- Camera Transform -->
	<rosparam param="p_C"> [0.0, 0.0, 0.15] </rosparam> 
	<rosparam param="q_O_C"> [-0.5, 0.5, -0.5, 0.5] </rosparam>

	<!-- Team Initilization -->
	<node name="coop_loc_init" pkg="coop_loc" type="team_init.py" output="screen">
		<param name= "init_time"  type= "double" value= "1"/>
	</node>

	<!-- Team Localization -->
	<node name="coop_loc" pkg="coop_loc" type="team_loc.py" output="screen" />

</launch>
